pkg/entrypoint/startup/startup.go

```diff
@@ -56,11 +56,9 @@ func runInitNetwork(ctx context.Context, log *logrus.Entry) error {
 		return err
 	}
 
-	if cs.Properties.PrivateAPIServer {
```

Comment:
I think this is because this function https://github.com/openshift/openshift-azure/blob/master/pkg/startup/v15/startup.go#L45 is stripping it out. An alternative solution is to add privateApiServer to that function.

Reply:
Does this makes this to be applied to all clusters?

Reply:
Testing this: 
https://github.com/openshift/openshift-azure/pull/2229